README.md

```diff
@@ -64,6 +64,14 @@ const client = new Client({
 })
 ```
 
+or from within a Pod:
+
+```js
+const Client = require('kubernetes-client').Client
+const config = require('kubernetes-client').config;
+const client = new Client({ config: config.getInCluster() });
+```
```

Comment:
you need to specify a version or, probably better, add a:

```js
await client.loadSpec();
```

at the end of this example.